src/scenes/ProgressReports/Detail/StatusStepper.tsx

```diff
@@ -19,13 +18,13 @@ import { ProgressReportStatus as Status } from '~/api/schema/schema.graphql';
 import { extendSx } from '~/common';
 
 type StatusStepperProps = {
-  current?: Status | null;
   loading: boolean;
+  current?: Status;
```

Comment:
This is what I was talking about on zoom yesterday. You made a change and then rolled it back but in a different way. This still results in a diff, but it's not actually changing anything.